org.eclipse.xtext/src/org/eclipse/xtext/xtext/XtextConfigurableIssueCodes.java

```diff
@@ -47,6 +47,7 @@
 	public static final String INVALID_ANNOTAION = ISSUE_CODE_PREFIX + "InvalidAnnotation";
 	public static final String USAGE_OF_DEPRECATED_RULE = ISSUE_CODE_PREFIX + "UsageOfDeprecatedRule";
 	
+	public static final String INVALID_FRAGMENT_AS_FIRST_RULE = ISSUE_CODE_PREFIX + "InvalidFragmentFirstRule";
```

Comment:
nitpick: why empty line before?

Reply:
Same as above.
Would you prefer to drop that empty line?

Reply:
Sorry, I did not see that this was already the case above in the file. Imitating the file's existing style is right. But when I see this correctly, the blocks are "errors" and "warnings". So this should go the "errors" section.

Reply:
Thanks for the hint! I will change that